packages/commons/src/repositories/ReadModelRepository.ts

```diff
@@ -9,6 +9,7 @@ import {
   genericInternalError,
   ProducerKeychainReadModel,
   ProducerJWKKey,
+  Delegation,
```

Comment:
As disscuss offline *Readmodel wrapper is not more usefull, in this case the new entity in ReadModel DB are not consumed by legacy scala services,so conversion to ISOstring is not required anymore, we can save directly as Date type.